src/current/v22.1/import-performance-best-practices.md

```diff
@@ -44,77 +105,67 @@ IMPORT INTO customers (id, name)
-
-`MYSQLDUMP` and `PGDUMP` run a single thread to parse their data, and therefore have substantially slower performance.
-
-`MYSQLDUMP` and `PGDUMP` are two examples of "bundled" data. This means that the dump file contains both the table schema and the data to import. These formats are the slowest to import, with `PGDUMP` being the slower of the two. This is because CockroachDB has to first load the whole file, read the whole file to get the schema, create the table with that schema, and then import the data. While these formats are slow, see [Import the schema separately from the data](#import-the-schema-separately-from-the-data) for guidance on speeding up bundled-data imports.
-
-{% include {{ page.version.version }}/import-table-deprecate.md %}
-
-### Import the schema separately from the data
+You can split the data into **more** files than you have nodes. CockroachDB will process the files in parallel across the cluster. When splitting the data Cockroach Labs recommends splitting it into a multiple of the number of nodes in your cluster, if possible. For example, if you have a 3 node cluster, split the dataset into 9, 27, or 300 files.
 
-For single-table `MYSQLDUMP` or `PGDUMP` imports, split your dump data into two files:
-
-1. A SQL file containing the table schema
-1. A CSV file containing the table data
+Cockroach Labs recommends keeping the files to a maximum file size of 4 GB unless the files are streamed (for example, [cloud storage locations](use-cloud-storage-for-bulk-operations.html) will stream the data to CockroachDB), and to keep each file size similar across the dataset. For example, if you are importing a 9 GB dataset that was split into 3 files into a 3 node cluster, keep each file around 3 GB in size if possible. Don't split the data into two 4 GB files, and one 1 GB file.
+{{site.data.alerts.end}}
 
-Then, import the schema-only file:
+For maximum performance each split file should be sorted within and across all files, meaning that if you were to sort the rows in each file there would be no overlapping data in any other file. For example, suppose your table has an alphabetic string primary key and you were importing the data into a 3 node cluster.
 
+{% include_cached copy-clipboard.html %}
 ~~~ sql
-> IMPORT TABLE customers
-FROM PGDUMP
-    'https://s3-us-west-1.amazonaws.com/cockroachdb-movr/datasets/employees-db/pg_dump/customers.sql' WITH ignore_unsupported_statements
-;
+CREATE TABLE contacts (email STRING PRIMARY KEY, first_name TEXT, last_name TEXT);
 ~~~
 
-And use the [`IMPORT INTO`](import-into.html) statement to import the CSV data into the newly created table:
+You should split the files so that the first file contains rows where the primary key begins with the letters "a" to "i," the second file "j" to "r," and the third file "s" to "z". No file contains duplicate rows, and within each file the data is sorted alphabetically by the primary key.
```

Comment:
```suggestion
You should split the files so that the first file contains rows where the [primary key](primary-key.html) begins with the letters "a" to "i," the second file "j" to "r," and the third file "s" to "z". No file should contain duplicate rows, and within each file the data is sorted alphabetically by the primary key.
```